interpreter/CMakeLists.txt

```diff
@@ -151,6 +151,11 @@ if(builtin_llvm)
   set(CMAKE_BUILD_TYPE ${LLVM_BUILD_TYPE})
   set(BUILD_SHARED_LIBS "NO")
 
+  if(CMAKE_BUILD_TYPE STREQUAL "Release" AND CMAKE_CXX_FLAGS_RELEASE STREQUAL "")
```

Comment:
I find "check the flags in CMakeCache.txt" not really helpful. IIUC this is a problem of incremental builds transitioning from before your `CXXFLAGS` changes to after. So "delete your CMakeCache.txt and reconfigure" might be better?

Should we create a blocking Jira ticket that this check should be removed for 6.20?

Reply:
I'm working on a thorough rewrite of the build flags, but after I finish the problem with no sources in older versions of CMake. After that, this will become unnecessary.

Reply:
In 
> this will become unnecessary
"this" refers to this PR? Then I'd suggest we close this PR?

Reply:
I will set `CMAKE_CXX_FLAGS` to `CMAKE_CXX_FLAGS_RELEASE` as a *non-cached* variable if it's empty. That won't change the cache, but will be used instead of it. If you then change the value in the cache, it will be used instead of the default flags.